pkg/conditions/controlplaneready/testdata/controlplane-with-ready-false.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: cluster.x-k8s.io/v1alpha3
+kind: Machine
```

Comment:
It's not a CR we actually use for the control plane, but we just need any CR to test stuff.